deptofdefense/move.mil
# Add Tutorials "show" view

## Checklist

I have…


## Summary of Changes

This pull request adds a "view all in single page" link to each tutorial on `localhost:3000/tutorials` which links to a new page for each tutorial (e.g. `localhost:3000/tutorials/foo-bar-biz-baz`). For "pretty" URLs, we're using [the FriendlyId gem](https://github.com/norman/friendly_id) and adding a `slug` column to the `tutorials` table that's automatically generated based on the tutorial's `title` when the record is saved.

## Testing

To verify the changes proposed in this pull request…

1. clone this repo,
1. git checkout `add-tutorials-show-view`,
1. set up development dependencies according to `CONTRIBUTING.md`,
1. `bundle install`,
1. `bin/rails db:migrate`,
1. `bin/rails server`, and
1. load up http://localhost:3000/tutorials in your Web browser of choice.

## Screenshots

## Narrow Viewport

![narrow](https://user-images.githubusercontent.com/27780860/28985270-bb64f9cc-792f-11e7-8c8f-2ef39f536aa1.png)

## Wide Viewport

![wide](https://user-images.githubusercontent.com/27780860/28985275-bf504b18-792f-11e7-8f1f-00b29f70890b.png)
